kinto/core/initialization.py

```diff
@@ -56,6 +56,19 @@ def setup_json_serializer(config):
     config.add_renderer("json", renderer)
 
 
+def setup_csp_headers(config):
+    """Content-Security-Policy HTTP response header helps reduce XSS risks on
+    modern browsers by declaring, which dynamic resources are allowed to load.
+    On APIs, we disable everything.
+    """
+    disable_all = "default-src 'none'; " "frame-ancestors 'none'; " "base-uri 'none'; "
```

Comment:
Why not just `"default-src 'none'; frame-ancestors 'none'; base-uri 'none';"`? Why the trailing space in the string?

Reply:
haha, that was `black` magic :)